CHANGELOG.md

```diff
@@ -21,6 +21,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     - prices of products are recalculated for logged customers based on custom discount
     - discount works also for companies with multiple users
     - `filterFormMacro.html.twig` now counts with discount coeficient
+- [#55 - Third domain with german locale](https://github.com/shopsys/demoshop/pull/55)
+    - domains and domains_url search index routing were added for third domain
+    - datafixtures were fixed to generate only supported multidomain demodata
+    - CategoryRepositoryTest was fixed for third domain
+    - translations were copied from en language and prefixed with (de)
```

Comment:
US says that prefix should be "DE_" why did you chose something else?

Reply:
@LukasHeinz changed his opinion today